src/server/lib/person.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+var fs = require("fs");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fs' is defined but never used.](https://app.codacy.com/app/Codacy-ACME/nodeproject/pullRequest?prid=4310063)